README.md

```diff
@@ -99,7 +99,7 @@ Desktop
 ```
 mkdir -p ros2_java_ws/src
 cd ros2_java_ws
-curl -skL https://raw.githubusercontent.com/esteve/ros2_java/master/ros2_java_desktop.repos -o ros2_java_desktop.repos
+curl -skL https://raw.githubusercontent.com/esteve/ros2_java/dashing/ros2_java_desktop.repos -o ros2_java_desktop.repos
```

Comment:
@nielstiben could you also update the URL to use the ros2-java organization? That is https://raw.githubusercontent.com/ros2-java/ros2_java/dashing/ros2_java_desktop.repos Thanks!

Reply:
@esteve, updated it to ros2-java organization repository.